src/control/nav2_dwb_controller/costmap_2d/include/costmap_2d/cost_values.h

```diff
@@ -1,7 +1,8 @@
-/*
+/*********************************************************************
```

Comment:
I didn't actually change the copyright header here. Git is just confused about where this file comes from. In actual fact, what it thinks is the original file was deleted, and this file was added.

You'll see more of this problem in the next few files. The reason is the same.

All the files in the costmap_2d folder here are from the actual costmap_2d with all the internal workings removed. All that is left is enough to allow compiling and linking. This whole directory will be deleted and replaced by the real costmap_2d package when it is ready.